test/test_arbitrage.py

```diff
@@ -1,9 +1,8 @@
 #!/usr/bin/env python
-
+import asyncio
 from os.path import join, realpath
-import sys; sys.path.insert(0, realpath(join(__file__, "../../")))
-
-from decimal import Decimal
+import sys;
+sys.path.insert(0, realpath(join(__file__, "../../")))
```

Comment:
If you're monkey patching something, the convention is to put it in the same line as the import, s.t. you don't trigger PEP8 violation warnings in PyCharm.